contributing/documentation/standards.rst

```diff
@@ -53,13 +53,22 @@ Code Examples
 * To avoid horizontal scrolling on code blocks, we prefer to break a line
   correctly if it crosses the 85th character;
 * When you fold one or more lines of code, place ``...`` in a comment at the point
-  of the fold. These comments are: ``// ...`` (php), ``# ...`` (yaml/bash), ``{# ... #}``
-  (twig), ``<!-- ... -->`` (xml/html), ``; ...`` (ini), ``...`` (text);
+  of the fold. These comments are:
+
+  * ``// ...`` (php)
+  * ``# ...`` (yaml/bash)
+  * ``{# ... #}`` (twig)
+  * ``<!-- ... -->`` (xml/html)
```

Comment:
maybe add a note that the HTML commit is also used when writing PHP template (i.e. the ``html+php`` format)